Automattic/wp-calypso
# Plugins: Update the ratings visualization on the cards

#### Proposed Changes

Update the ratings display on the Plugins Browser cards.

#### Testing Instructions
* Go to `/plugins`
* Check the ratings on the plugins that contains ratings
* Search for something and check if the .org plugins have ratings

| Before  | After |
| ------------- | ------------- |
|<img width="1072" alt="CleanShot 2023-01-20 at 13 07 32@2x" src="https://user-images.githubusercontent.com/5039531/213763077-ddff11a3-9f7c-45bb-86b1-1f5edd28ceba.png">|<img width="1053" alt="CleanShot 2023-01-20 at 13 12 13@2x" src="https://user-images.githubusercontent.com/5039531/213763119-3df5a93f-8ecd-48e9-a1dd-2f8b07b6e43f.png">|


#### Pre-merge Checklist





Fixes #70906
